src/PIL/PngImagePlugin.py

```diff
@@ -696,8 +696,14 @@ def load_end(self):
     def _getexif(self):
         if "exif" not in self.info:
             self.load()
-        from .JpegImagePlugin import _getexif
-        return _getexif(self)
+        if "exif" not in self.info:
+            return None
+        return dict(self.getexif())
+
+    def getexif(self):
```

Comment:
🚲🏡time: I know #520 suggested it, but is `getexif` of `get_exif` the better name?

Reply:
@hugovk Depends on what you mean by "better"? If we have some convention of using ``get_`` everywhere, then the latter. Else, perhaps the former. Personally, the first one reads better to me but I'm not entirely sure why.

Reply:
In this class, there's a `getchunks`. Across the codebase there's more `getx` than `get_x`. Let's keep this :)